### PR TITLE
node.js: upgrade OpenCensus and Zipkin exporter to 0.0.19

### PR DESCRIPTION
**Pull request by alexamies**
II had problems with the older version on my Mac laptop although it worked ok on my more Linux desktop. This is resolved in the latest version.

### CODE REVIEW

**Review by mayurkale22 — APPROVED** [review]
lgtm

File: node.js/package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
-    "@opencensus/exporter-zipkin": "0.0.7",
-    "@opencensus/nodejs": "0.0.7"
+    "@opencensus/exporter-zipkin": "0.0.19",
```
Comment (mayurkale22) [inline node.js/package.json]: Perhaps we should use caret version (`^`): will update you to the next minor version. `0.0.19` => `^0.0.19`.

Comment (alexamies) [reply node.js/package.json]: Changed

**Review by odeke-em — APPROVED** [review]
LGTM, thank you @alexamies for the PR and @mayurkale22 for the review!

**Pull request merged**